web/app.js

```diff
@@ -187,6 +188,7 @@ const PDFViewerApplication = {
   url: "",
   baseUrl: "",
   externalServices: DefaultExternalServices,
+  annotationStorage: new AnnotationStorage(),
```

Comment:
Assuming that the `annotationStorage` is document-specific, you'll have to initialize this for each file opened in the viewer rather than globally like this (since the default viewer supports opening more than the initial document).

Reply:
Good catch: I didn't know that... I'm a kind of newbie.